### DISCUSSION TITLE
Added FUNDING.yml to uuidjs/uuid, but not seeing "Sponsor this project" in the sidebar

### DISCUSSION THREAD

**Discussion by broofa** [community/community#37893] [Other Feature Feedback, Questions, & Ideas] [answered]
As the title says, I just added a `.github/FUNDING.yml` file to [uuidjs/uuid](https://github.com/uuidjs/uuid), but I'm not seeing the expected "Sponsor this project" section in the sidebar.

... what makes this even more unexpected is I did exactly the same thing for another one of my org/repos - [npmgraph/npmgraph](https://github.com/npmgraph/npmgraph) - and it worked fine.

Anyone know what the problem might be?

[Edit to add: `uuidjs` is not set up with an organization account for sponsorships, but neither is `npmgraph`.  I believe I have both accounts/repositories set up similarly, with a FUNDING.yml file that names two individual accounts.  So... not really sure why the latter works and the former doesn't.]

**Comment by antonblr** [comment, marked as answer]
Hi @broofa - for the sake of completeness could you add screenshots of the **Settings > Features - Sponsorships** details from both repos (as seen in [Displaying a sponsor button in your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository#displaying-a-sponsor-button-in-your-repository)). Other than `.github/FUNDING.yml` files setup, this setting seems to be the only possible reason.

And FWIW, I played a bit with [uuidjs/uuid](https://github.com/uuidjs/uuid) `.github/FUNDING.yml` and got:

<img width="538" alt="image" src="https://user-images.githubusercontent.com/44246099/199066669-06e2a201-ec0d-4987-8d2e-b7ad89dde39d.png">


_EDIT_: Something going on with `ctavan` user. I see ^ this error when I leave this user only in the list. 

_EDIT2_: Make sure `ctavan` has setup https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account

**Reply by broofa** [reply]
~~@antonblr What happens if you have just `broofa` in the list?   I tried removing `ctavan` so I was the only one in the FUNDING list  (prior to posting here) and it didn't seem to fix the problem.~~

Edit: Nevermind... following your suggestion, it turns out I didn't have the "Display a "Sponsor" button" option turned on in the repo settings.  🤦   Thanks for the help, and sorry for the trouble!

**Reply by DhavalGojiya** [reply]
@broofa 
I am facing the same issue in my repo: [setup-solr-action](https://github.com/DhavalGojiya/setup-solr-action)
The sponsor button is not showing even after adding the `.github/FUNDING.yml` file.

**Reply by rfgamaral** [reply]
@DhavalGojiya I see the sponsor button in your project, what did you do to solve it? I'm having the same issue.

**Reply by DhavalGojiya** [reply]
> @DhavalGojiya I see the sponsor button in your project, what did you do to solve it? I'm having the same issue.

I just resolve the issue.
Do this :
Go to Repository homepage > settings > enable sponsor button
